_layouts/ballerina-why-ballerina-left-nav-pages-swanlake.html

```diff
@@ -0,0 +1,98 @@
+                     </div>
+                     <div class="col-md-12 col-lg-2 cBallerina-io-breadcrumbs cGitLink">
+                       <a class="icon icon-github" target="_blank" href="https://www.github.com/{{ site.github_username }}/{{ site.git_repo }}/blob/master/{{ page.path | replace: ".html", ".md"}}"></a>
+                     </div>
+                  </div>
+                  <div class="cBlallerina-io-docs-content-container cPositionRelative">
+                     <div class="wy-nav-content">
+                        <div class="rst-content">
+                           <div role="main">
+                              <div class="section">
+                                 <h1>{{ page.title }}</h1>
+                                 <p>{{ page.intro }}</p>
+                                 <div class="cBallerinaTocContainer" id="table-of-content">
+                                    <p class="cTOCcontol" ><strong>Table of contents</strong>  &nbsp;  <a href="#" id="tree-expand-all">Expand all</a> &nbsp; <a href="#" id="tree-collapse-all">Collapse all</a></p>
+                                    {% include toc.html html=content sanitize=true id="tree" %}
+                                 </div>
+                                 {{ content }}
+                              </div>
+                           </div>
+                        </div>
+                     </div>
+                     <a class="cTopLink" href="#top">Back to Top &#x2934;</a>
+                  </div>
+               </div>
+            </div>
+         </div>
+      </div>
+      {% include ballerina-footer.html %}
+      <script src="/js/theme.js"></script>
+      <script>
+         $(document).ready(function () {
+            var gitstar = '<iframe id="iGitStarIframe" src="https://ghbtns.com/github-btn.html?user=ballerina-platform&repo=ballerina-lang&type=star&count=true&size=small" frameborder="0" scrolling="0"></iframe>';
+            $('[data-button="iGitStarText"]').replaceWith(gitstar);
+         
+            var gitwatch = '<iframe id="iGitWatchIframe" src="https://ghbtns.com/github-btn.html?user=ballerina-platform&repo=ballerina-lang&type=watch&count=true&size=lsmalle&v=2" frameborder="0" scrolling="0"></iframe>';
+            $('[data-button="iGitWatchText"]').replaceWith(gitwatch);
+         
+         });
+         
+     </script>
+     <script src="/js/ballerina-leftnav.js"></script>
+
+
+
+     <script src="https://code.jquery.com/jquery-1.11.0.min.js"></script>
+     <script src="/js/abixTreeList.min.js"></script>
+     <script>
+         $(document).ready(function() {
+             $('#tree').abixTreeList();
+         });
+
+
+
+
+
+         $(window).scroll(function () {
+        var bodyclass = document.body;
+
+        if ($(this).scrollTop() > 200) {
+            $('.cTopLink').removeClass('dispalaynone');
+          
+        } else {
+            $('.cTopLink').addClass('dispalaynone');
+          
+        }
+
```

Comment:
@praneesha These unnecessary empty lines can be removed?

Reply:
Fixed.